src/Search.php

```diff
@@ -7257,15 +7256,20 @@ public static function giveItem(
                             $name = sprintf(__('%1$s (%2$s)'), $name, $data[$ID][0]['id']);
                         }
                         $out    .= $name . "</a>";
+
+                        // Add tooltip
+                        $id = $data[$ID][0]['id'];
+                        $itemtype = getItemTypeForTable($table);
                         $out     = sprintf(
                             __('%1$s %2$s'),
                             $out,
                             Html::showToolTip(
-                                RichText::getEnhancedHtml($data[$ID][0]['content']),
+                                __('Loading...'),
                                 [
                                     'applyto'        => $itemtype . $data[$ID][0]['id'],
                                     'display'        => false,
-                                    'images_gallery' => false, // don't show photoswipe gallery in tooltips
+                                    'images_gallery' => false,  // don't show photoswipe gallery in tooltips
```

Comment:
IMHO, this option was meant to be passed to the `RichText::getEnhancedHtml()` method. It should be moved into the `ajax/get_item_content.php` file.

Reply:
Indeed, this parameter was sent to the wrong function.
Since false is the default value, it did not seems to cause any unwanted behavior, I'll still set it in the new endpoint just in case.